src/throttle.cpp

```diff
@@ -43,42 +67,30 @@ uint16_t Throttle::ReadAcceleratorPress()
 
     //right_acc_val, left_acc_val, brake_val are exact values from sensor (analogRead)
     //right_acc_pos, left_acc_pos, brake_pos will be a value from 0 to 100 -> allows for comparison because sensors have different ranges
-    uint16_t right_acc_val = max(analogRead(acc_sensor_right), MIN_VAL_RIGHT); //Don't get negative values
+    uint16_t right_acc_val = max(analogRead(ACC_SENSOR_RIGHT), MIN_VAL_RIGHT); //Don't get negative values
     right_acc_pos = (right_acc_val-MIN_VAL_RIGHT)*100/(MAX_VAL_RIGHT-MIN_VAL_RIGHT);
     /*
-    uint16_t left_acc_val = max(analogRead(acc_sensor_left), MIN_VAL_LEFT); 
+    uint16_t left_acc_val = max(analogRead(ACC_SENSOR_LEFT), MIN_VAL_LEFT); 
     left_acc_pos = (left_acc_val-MIN_VAL_LEFT)*100/(MAX_VAL_LEFT-MIN_VAL_LEFT);
-    uint16_t brake_val = max(analogRead(brake_sensor), MIN_VAL_BRAKE); 
+    uint16_t brake_val = max(analogRead(BRAKE_SENSOR), MIN_VAL_BRAKE); 
     brake_pos = (brake_val-MIN_VAL_BRAKE)*100/(MAX_VAL_BRAKE-MIN_VAL_BRAKE);
     */
 
     /* When we have all 3 sensors:
     if (!brakeAndAccelerator() && arePotentiometersCorrect()) {
-        float throttle_percent = (right_acc_pos + left_acc_pos)/2;
+        float throttle_percent = (right_acc_pos + left_acc_pos)/2.0;
     }
     else {
         return 0;
     }
     */
 
-    //for now
+    // for now
     float throttle_percent = right_acc_pos;
 
-    //apply equation to convert the throttle's position (percent) to torque
-    uint16_t torque = min(exp(0.06 * (throttle_percent - 9)), 230.0);
-
-    /*
-    Serial.print("Voltage (FOR TESTING): ");
-    Serial.println(pedal_val);
-    Serial.print("Throttle Percent: ");
-    Serial.println(throttle_percent);
-    Serial.println();
-    Serial.print("Torque: ");
-    Serial.println(torque);
-    Serial.println();
-    */
-    return torque;
-};
+    // Set throttle position sensor value (0-1)
+    throttle_perc = throttle_percent/100.0;
+}
 
 bool Throttle::arePotentiometersCorrect()
```

Comment:
if this bool is false, we want to send a tx message with torque 0 asap, so manually trigger a tx message with the EncodeAndSend method